syntax = "proto3";

option go_package = "github.com/mkacz91/spejs/pb";

service UniverseService {
  rpc Ping (PingRequest) returns (PingResponse) {}
}

message PingRequest {
  string message = 1;
}

message PingResponse {
  string message = 1;
}